core/src/autogluon/core/features/infer_types.py

```diff
@@ -136,3 +136,26 @@ def check_if_nlp_feature(X: Series) -> bool:
         return False
 
     return True
+
+
+def get_bool_true_val(series: pd.Series, replace_val=None):
+    """
+    From a pandas series, get the replace_val to convert to boolean when calling:
+    series_bool = series == replace_val
+
+    series must have exactly 2 unique values
+    """
+    if replace_val is None:
+        try:
+            uniques = series.unique()
+            uniques.sort()
```

Comment:
maybe add a comment that this is a check for mixed types present in series

Reply:
added comments, it is for more than checking for mixed types. `.sort()` updates in-place, which allows consistency in which value is chosen as `True`.